www/src/components/repos/queries.js

```diff
@@ -195,7 +195,7 @@ export const CHART_Q = gql`
       }
       installation { ...ChartInstallationFragment }
     }
-    versions(chartId: $chartId, first: 10, after: $cursor) {
+    versions(chartId: $chartId, first: 15, after: $cursor) {
```

Comment:
I'll switch back to 10 once the button to load more will be done. 100 cannot be used because it scroll doesn't show inside select panel.